components/multiselect/multiselect.jsx

```diff
@@ -175,6 +175,13 @@ export default class MultiSelect extends React.Component {
         this.props.handleSubmit();
     }
 
+    handleSubmitKeyDown = (e) => {
+        if (e.keyCode === KeyCodes.SPACE[1]) {
```

Comment:
I am not sure if this will hold true if we consider diff layout and language keyboard solution. Can you check for keyPress based off of `isKeyPressed` Util func in utils/utils.jsx?.

Reply:
Done!